synapse/handlers/room_member.py

```diff
@@ -1104,32 +1104,34 @@ async def remote_reject_invite(
             #
             logger.warning("Failed to reject invite: %s", e)
 
-            return await self._locally_reject_invite(
+            return await self._generate_local_out_of_band_membership(
                 invite_event, txn_id, requester, content
             )
 
-    async def _locally_reject_invite(
+    async def _generate_local_out_of_band_membership(
```

Comment:
```suggestion
    async def _generate_local_out_of_band_leave(
```

?

Reply:
Yes, it's not any ol' membership is it :D